packages/suite/src/components/suite/Banners/index.tsx

```diff
@@ -58,6 +64,19 @@ const Banners = () => {
     } else if (device?.features?.needs_backup) {
         banner = <NoBackup />;
         priority = 7;
+    } else if (device?.connected && device?.features?.safety_checks === 'PromptAlways') {
+        // PromptAlways could only be set via trezorctl. Warn user unconditionally.
+        banner = <SafetyChecksBanner />;
+        priority = 6;
+    } else if (
+        !safetyChecksDismissed &&
+        device?.connected &&
+        device?.features?.safety_checks === 'PromptTemporarily'
+    ) {
+        // PromptTemporarily was probably set intentionally via Suite and will change back to Strict when Trezor reboots.
+        // Let the user dismiss the warning.
+        banner = <SafetyChecksBanner onDismiss={() => setSafetyChecksDismissed(true)} />;
+        priority = 6;
```

Comment:
```suggestion
        priority = 50;
```

Reply:
as well adressed in the rebase on your branch
https://github.com/trezor/trezor-suite/compare/fdab83a36049fdd373e8d9391c7b1dc946b33769..94386fd286a4f177e2508aa356dc4f3df0b5baf6